### PR TITLE
Babel 7.14.0 blog post

### PR DESCRIPTION
**Pull request by nicolo-ribaudo**
Preview: https://deploy-preview-2508--babel.netlify.app/blog/2021/04/26/7.14.0

### CODE REVIEW

File: website/blog/2021-04-26-7.14.0.md [hunk 1]
```diff
@@ -0,0 +1,144 @@
...
+    return this.#value * 2;
+  }
+}
+```
+
+You can remove `@babel/plugin-proposal-class-properties` and `@babel/plugin-proposal-private-methods`, since they are now enabled by default in `@babel/preset-env`.
+
+If you are using `@babel/preset-env`'s [`shippedProposals`](https://babeljs.io/docs/en/babel-preset-env#shippedproposals) option, it now also includes the `@babel/plugin-proposal-private-property-in-object` and `@babel/plugin-proposal-class-static-block` plugins: you can safely remove them from your configuration.
+
+> ⚠️ Webpack [doesn't support](https://github.com/webpack/webpack/issues/10216) this syntax yet.
```
Comment (sosukesuzuki) [inline website/blog/2021-04-26-7.14.0.md:51]: Now webpack supports class fields since 5.36.0 https://github.com/webpack/webpack/releases/tag/v5.36.0

Comment (hzoo) [reply website/blog/2021-04-26-7.14.0.md]: Added it!!

**Review by existentialism — APPROVED** [review]

**Review by hzoo — APPROVED** [review]

File: website/blog/2021-04-26-7.14.0.md [hunk 2]
```diff
@@ -0,0 +1,165 @@
...
+}
+
+console.log(sync());
+// Logs:
+// - "First!"
+// - "Second!"
+// - "Third!"
+```
+
+You can test this proposal (and report feedback!) by adding the [`@babel/plugin-proposal-do-expressions`](https://babeljs.io/docs/en/babel-plugin-proposal-do-expressions) and `@babel/plugin-proposal-async-do-expressions` plugins to your Babel configuration.
```
Comment (hzoo) [inline website/blog/2021-04-26-7.14.0.md:163]: Was going to link this if we make a doc for it `@babel/plugin-proposal-async-do-expressions`

Comment (nicolo-ribaudo) [reply website/blog/2021-04-26-7.14.0.md:163]: https://github.com/babel/website/pull/2486

```suggestion
You can test this proposal (and report feedback!) by adding the [`@babel/plugin-proposal-do-expressions`](https://babeljs.io/docs/en/babel-plugin-proposal-do-expressions) and [`@babel/plugin-proposal-async-do-expressions`](https://babeljs.io/docs/en/babel-plugin-proposal-async-do-expressions) plugins to your Babel configuration.
```

**Comment by dilyanpalauzov** [comment]
The blogpost states, that @babel/plugin-proposal-class-properties and @babel/plugin-proposal-private-methods are implicit for  @babel/preset-env.

The announcement is unclear, whether the same applies for @babel/preset-typescript, for the case where babel is only used to strip TS type annotations?

**Comment by nicolo-ribaudo** [comment]
For TS, they are parser be default but not transformed to es6 class syntax.

**Comment by dilyanpalauzov** [comment]
Yes, they are parsed but not transformed.  In previous babel versions they were not parsed by default.

**Pull request merged**